src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -2,62 +2,75 @@
+            .ForwardAs("--deprecated");
+
+        public static readonly Option VulnerableOption = new Option<bool>("--vulnerable", LocalizableStrings.CmdVulnerableDescription)
+            .ForwardAs("--vulnerable");
+
+        public static readonly Option FrameworkOption = new Option<IEnumerable<string>>("--framework", LocalizableStrings.CmdFrameworkDescription)
+        {
+            Argument = new Argument<IEnumerable<string>>(LocalizableStrings.CmdFramework) { Arity = ArgumentArity.OneOrMore }
+        }.ForwardAsMany(o => ForwardedArguments("--framework", o));
+
+        public static readonly Option TransitiveOption = new Option<bool>("--include-transitive", LocalizableStrings.CmdTransitiveDescription)
+            .ForwardAs("--include-transitive");
+
+        public static readonly Option PrereleaseOption = new Option<bool>("--include-prerelease", LocalizableStrings.CmdPrereleaseDescription)
+            .ForwardAs("--include-prerelease");
+
+        public static readonly Option HighestPatchOption = new Option<bool>("--highest-patch", LocalizableStrings.CmdHighestPatchDescription)
+            .ForwardAs("--highest-patch");
+
+        public static readonly Option HighestMinorOption = new Option<bool>("--highest-minor", LocalizableStrings.CmdHighestMinorDescription)
+            .ForwardAs("--highest-minor");
+
+        public static readonly Option ConfigOption = new Option<string>("--config", LocalizableStrings.CmdConfigDescription)
+        {
+            Argument = new Argument<string>(LocalizableStrings.CmdConfig)
+        }.ForwardAsMany(o => new[] { "--config", o });
+
+        public static readonly Option SourceOption = new Option<IEnumerable<string>>("--source", LocalizableStrings.CmdSourceDescription)
+        {
+            Argument = new Argument<IEnumerable<string>>(LocalizableStrings.CmdSource) { Arity = ArgumentArity.OneOrMore }
+        }.ForwardAsMany(o => ForwardedArguments("--source", o));
+
+        public static readonly Option InteractiveOption = new Option<bool>("--interactive", CommonLocalizableStrings.CommandInteractiveOptionDescription)
+            .ForwardAs("--interactive");
+
+        public static Command GetCommand()
+        {
+            var command = new Command("package", LocalizableStrings.AppFullName);
+
+            command.AddOption(CommonOptions.VerbosityOption(o => $"--verbosity:{o}"));
+            command.AddOption(OutdatedOption);
+            command.AddOption(DepreciatedOption);
+            command.AddOption(VulnerableOption);
+            command.AddOption(FrameworkOption);
+            command.AddOption(TransitiveOption);
+            command.AddOption(PrereleaseOption);
+            command.AddOption(HighestPatchOption);
+            command.AddOption(HighestMinorOption);
+            command.AddOption(ConfigOption);
+            command.AddOption(SourceOption);
+            command.AddOption(InteractiveOption);
+            command.AddOption(CommonOptions.DiagOption());
```

Comment:
The current `dotnet list package` command doesn't have `--diagnostics` option. No references to this option in NuGet side [implementation](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs) for this command. I see that this is one of the common options. Is this something new that we should support?